modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDisplayPageTemplates.testcase

```diff
@@ -886,6 +886,125 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9392. Verify that Related Diagrams are available for use with Collection Display Fragment."
+	@priority = 5
+	test CanMapRelatedDiagramsToCollectionDisplayFragment {
+		property portal.acceptance = "true";
+
```

Comment:
@alessiorendina can you please add this here: 
`property test.run.environment = "EE";`
so we can prevent this test from running on CE environments. Thanks!

Reply:
Sure